src/rider/main/kotlin/me/seclerp/rider/plugins/efcore/components/items/TargetFrameworkItem.kt

```diff
@@ -2,6 +2,14 @@ package me.seclerp.rider.plugins.efcore.components.items
 
 import me.seclerp.rider.plugins.efcore.DotnetIconResolver
 import me.seclerp.rider.plugins.efcore.DotnetIconType
+import javax.swing.Icon
 
-class TargetFrameworkItem(displayName: String)
-    : IconItem<Unit>(displayName, DotnetIconResolver.resolveForType(DotnetIconType.TARGET_FRAMEWORK), Unit)
\ No newline at end of file
+class TargetFrameworkItem : IconItem<Unit> {
+    constructor(displayName: String, icon: Icon?) : super(displayName, icon, Unit)
```

Comment:
It's better to introduce such hierarchy here:
- `BaseTargetFrameworkItem`: abstract base class, accepts `displayText`, `icon` (nullable) and `data` (nullable)
- `TargetFrameworkItem`: which accepts `displayText` and `data` (non-nullable)
- `DefaultTargetFrameworkItem`: which accepts nothing